LemonUI/Scaleform/Celebration.cs

```diff
@@ -0,0 +1,502 @@
+﻿using System;
+using System.Collections.Generic;
+#if FIVEM
+using CitizenFX.Core.Native;
+#elif RPH
+using Rage.Native;
+#elif (SHVDN2 || SHVDN3)
+using GTA.Native;
+#endif
+
+namespace LemonUI.Scaleform
+{
+    /// <summary>
+    /// Celebration screen that is shown at the end of a mission or job.
+    /// </summary>
+    /// <footer>
+    /// <a href="https://www.google.com/search?q=LemonUI.Scaleform.Celebration">`Celebration` on google.com</a>
```

Comment:
Why the Google link? I feel that it serves no purpose.

Reply:
I added the Google links because it was done in some other classes too. I'll remove them